SPECK/src/main/java/SPEck.java

```diff
@@ -227,19 +296,22 @@ protected static void writeDataset(String fileOut, Utils.Dataset dataset) throws
     /**
      * Compute the p-values of the FSP of a dataset merging the p-values computed by the different cores.
      * @param   T               the number of random datasets for the Monte Carlo estimate of p-values
-     * @param   parallelization the number of cores of the machine. It is the number of random datasets
+     * @param   numCores        the number of cores of the machine. It is the number of random datasets
      *                          generated at the same time in the Monte Carlo estimation. Each core of
-     *                          the machine computes the p-values on T/parallelization random datasets.
+     *                          the machine computes the p-values on T/numCores random datasets.
      * @param   fileMined       the file containing the FSP extracted from the dataset stored in file
```

Comment:
Let's call it `fileFSP`, and describe it as the file containing the FSP with respect to the datasets "on which" we are computing the p-values (it may be the observed dataset or one of the P "Westfall-Young" datasets)